TENDERMINT_INTEGRATION.rst

```diff
@@ -0,0 +1,39 @@
+**********************
+Tendermint Integration
+**********************
```

Comment:
the guide does not run out of the box, you need to run `docker-compose -f docker-compose.tendermint.yml up --build`

Reply:
are you sure, in my experience the `up` command will run the build if needed

Reply:
my bad. It was a problem at my end.